### PR TITLE
[RfR] Prereg tooltip explaining required for all fields

### PR DESCRIPTION
**Pull request by barbour-em**
## Purpose

Tell users what to do for a required field that is not applicable to their study

## Changes 

Required questions have a beautiful tooltip

## Side effects

Just tooltips

[OSF-5329]

### CODE REVIEW

File: website/templates/project/registration_editor_templates.mako [hunk 1]
```diff
@@ -4,9 +4,18 @@
 </script>
 
 <script type="text/html" id="text">
-  <input data-bind="valueUpdate: 'keyup',
-                    value: value"
-         type="text" class="form-control" />
+    <span data-bind="if: required">
+        <input data-bind="valueUpdate: 'keyup',
+                          value: value,
+                          tooltip: {title: 'This field is required for submission. If this field is not applicable to your study, you may state so.', placement: 'right'}"
```
Comment (samanehsan) [inline website/templates/project/registration_editor_templates.mako]: I'm not sure I understand what part of the form this corresponds to. I see the tooltip defined below when I hover or click on a text box, but I'm not sure where this one is happening.

**Comment by samanehsan** [comment]
Minor detail, but when I have one textbox selected and I hover over another, I see two tooltips. Is this intended behavior? 
screen shot 2015-12-09 at 4 39 47 pm

**Comment by samanehsan** [comment]
Also, when I click on a textbox so that the tooltip appears, and then scroll down, the tooltip overlaps with the project nav bar:  
screen shot 2015-12-09 at 4 43 09 pm

**Comment by samanehsan** [comment]
Pass complete @barbour-em! :sunny:

**Comment by samanehsan** [comment]
Also, for pull requests that make changes to the UI please add screenshots to the PR description! It makes it so much easier to know what changed.

**Comment by sloria** [comment]
This has been implemented on develop and 0.61.0

**Pull request closed without merging**